public/app/plugins/panel/heatmap/HeatmapTooltip.tsx

```diff
@@ -99,9 +104,7 @@ const HeatmapHoverCell = ({
   const index = dataIdxs[1]!;
   const data = dataRef.current;
 
-  const [isSparse] = useState(
-    () => data.heatmap?.meta?.type === DataFrameType.HeatmapCells && !isHeatmapCellsDense(data.heatmap)
-  );
+  const [isSparse] = useState(() => isHeatmapSparse(data.heatmap));
```

Comment:
what in tarnation? 😆 I get this was like this before, so let's not change it in case there's some weird reason for it to work this way, but I figure we actually want a `useMemo` here.